archunit/src/test/java/com/tngtech/archunit/library/dependencies/EdgeTest.java

```diff
@@ -11,19 +14,21 @@
     @Test
     public void edges_are_equal_iff_from_and_to_are_equal() {
 
-        assertThat(new Edge<>(from, to)).isEqualTo(new Edge<>(from, to));
-        assertThat(new Edge<>(from, to)).isNotEqualTo(new Edge<>(new Object(), to));
-        assertThat(new Edge<>(from, to)).isNotEqualTo(new Edge<>(from, new Object()));
+        assertThat(newEdge(from, to)).isEqualTo(newEdge(from, to));
```

Comment:
I find calling the constructor more expressive here.

Reply:
Hmm, I'm ambiguous here. Because then you now have to specify `new Edge<>(from, to, Collections.emptySet())` or similar for every edge. Seems to also obscure a little what is the essence of the test?

Reply:
But maybe you are right, in particular in the second part of the test, where `newEdge` is compared to `new Edge(..)`. Okay, convinced, gonna inline it there and move the method :wink:

Reply:
Done